src/config.ts

```diff
@@ -26,7 +28,29 @@ export async function getAwsConfigFromOptionsOrFile(options: {
   region: string;
 }): Promise<AWSConfig> {
   const { profile, accessKey, secretKey, sessionToken, region } = options;
+  if (profile){
+    const credentials = await fromSSO({
+      profile: profile,
+    });
 
+    const sso_credentials = new SSOClient({
+      credentials: credentials,
+      region: region,
+    });
+    const creds = await sso_credentials.config.credentials()
```

Comment:
There seem to be typescript errors here:

<img width="634" alt="image" src="https://github.com/kamranahmedse/aws-cost-cli/assets/4921183/46c8feb3-f92d-4d9f-a0d4-252221125a22">

Also, let's please be consistent with the naming conventions. Camel case instead of underscores and `credentials` instead of `creds`.

Reply:
Camel case is now used instead of underscores and creds is replaced by credentials